content/docs/start/index.md

```diff
@@ -54,31 +54,44 @@ $ git commit -m "Initialize DVC"
 
 Now you're ready to DVC!
 
-The value of DVC's several feature sets is best understood from different
-angles. Pick one of the two trails below to learn about DVC from that
-perspective:
+## Following This Guide
 
-## Data
+To help you understand and use DVC better, consider those two high level
+scenarios:
 
-- **[Data and model versioning]** is the base layer of DVC for large files,
-  datasets, and machine learning models. Use a standard Git workflow, but
-  without storing large files in the repo. Data is cached by DVC, allowing for
-  efficient sharing. Think "Git for data".
+- **Data Management** - Track and version large amounts of data along with your
+  code, and use DVC as a build-system for reproducible, data driven pipelines.
 
-- **[Data and model access]** goes over using data artifacts from outside of the
-  project and importing them from another DVC project. This can help to download
-  a specific version of an ML model to a deployment server or import a dataset
-  into another project.
+- **Experiment Management** - Live-track your experiments via code
+  instrumentation for light-weight experiment tracking. Learn how to Leverage
+  pipelines as a powerful Git based build system with native parameters, metrics
+  and plots support.
```

Comment:
I took another stab at it by modifying your suggestion a bit

Reply:
My 2cs - I would simplify it. When I see Live it forces me to think if there is some special about it? Is it _only_ Live, is there non-live, etc.

Reply:
Thanks @omesser! This feels worth discussing since it is the first intro to experiments and also can help with our overall messaging.

> The dvclive "values" I think we want to highlight are "live" and "works by instrumenting your code" 😅 maybe we're trying to highlight different things. To explain, the reason I'm trying to say those 2 things is that those are 2 properties/features that competitor tools have and I believe users will understand.

Yeah, I think we have different goals in mind. My thoughts are that since users already likely have familiarity with those properties of other experiment trackers, they are implied and don't explain why to use DVC. If I'm new to DVC, it wouldn't be surprising that I can track experiments using code instrumentation, it would be surprising that DVC records all that in Git.

I can see how mentioning Git could be confusing without going into too much detail here, so this part is not a blocker to me.

> I've seen it refer to as a build tool / system (e.g. "makefile for ML projects") - I think that works well. more accurately a build tool than an orchestrator, but there's a drawback - "makefile" and "build system" might not resonate well with our target audience, but more with hardcore SW engineers.

We already mention it as a build tool in data management. My "orchestrator" suggestion was trying to separate data pipelines (auto build and skip cached stages) and experiment pipelines (queue and execute many variations). Maybe orchestrator isn't right, but I think that `dvc exp run --queue --set-param lr=range(0.01,0.1,0.01) --set-param opt=adam,sgd` goes beyond a build tool.

I also worry that both this paragraph and the reordering of the pages makes the collaboration seem dependent on the pipelines, which I think was why @daavoo had put that ahead of pipelines before. Overall, I think @daavoo has a point that it may be better ultimately to separate pipelines than explain them in both data management and experiment contexts. 

To focus on the main value prop of experiments, I'd be in favor of dropping pipelines from this paragraph for now and shortening to something like:

> Easily track your experiments and their progress by only instrumenting your code, and collaborate on ML experiments like software engineers do for code.

Reply:
Thanks for the explanation Dave!
It's good to hear more opinions here.

This goes into tangent a bit but:

> Maybe orchestrator isn't right, but I think that dvc exp run --queue --set-param lr=range(0.01,0.1,0.01) --set-param opt=adam,sgd goes beyond a build tool.

For the record, I would still prefer "build" tool or "job-runner" here. we're going with build tool for now everywhere else, so it's ok to stick with this imo.
Orchestration of workloads (like jobs, tasks, experiments) beyond that usually - things like allocation of jobs to resources (machines or specific cpu/gpu/mem allocation), auto-healing, moving jobs around to optimize according to different cluster/workload parameters... usually this supports integration with monitoring for the sake of more advanced execution scenarios, etc...  all those are wayyy out of scope. I do think we're dealing with "makefile" type task runner on steroids (because, pipelines).


> To focus on the main value prop of experiments, I'd be in favor of dropping pipelines from this paragraph for now and shortening to something like:
Yep, thanks for "compromising" here. If we go as far as making a decision to isolate pipelines it will expand the scope a lot here. so those short-short descriptions need to be super concise (and remember, when/if we move pipelines out, and I won't block that, this can be adjusted)

Reply:
Makes sense, thanks @omesser! Totally agree about keeping it concise here, so I guess we are agreed to drop the pipelines part of this paragraph and call this resolved?